src/hooks/useTechnology.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {
+  useState, createContext, useContext, useEffect, useCallback,
+} from 'react';
+import { getTechnology, getTechnologyCosts } from '../services/technology';
+
+const TechnologyContext = createContext({});
```

Comment:
Following the suggestion above, here you will need to change this:
```suggestion
const TechnologyContext = createContext<Technology | null>(null);
```

Reply:
Done